v3/src/styles/modal-style.ts

```diff
@@ -0,0 +1,41 @@
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react'
+
+// This function creates a set of functions that help us create multipart component styles.
+const helpers = createMultiStyleConfigHelpers(["modal", "header", "body", "button"])
+
+export const Modal = helpers.defineMultiStyleConfig({
+  baseStyle: {
+    header: {
+      font: "museo-sans, sans-serif",
```

Comment:
I think this will look for a single font with the name `museo-sans, sans-serif`. The built-in string doesn't need quotes, so what you want is:
```suggestion
      font: "museo-sans", sans-serif,
```